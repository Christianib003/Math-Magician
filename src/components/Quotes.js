import React from 'react';

const Quotes = () => (
  <>
    <div className="quotes">
      <p className="qtext">
        1. “The purpose of our lives is to be happy.” —
        <span> Dalai Lama</span>
      </p>
      <p className="qtext">
        2. “Get busy living or get busy dying.” —
        <span> Stephen King</span>
      </p>
      <p className="qtext">
        3. “You only live once, but if you do it right, once is enough.” —
        <span> Mae West</span>
      </p>
    </div>
  </>
);

export default Quotes;
