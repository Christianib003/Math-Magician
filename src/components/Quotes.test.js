import React from 'react';
import renderer from 'react-test-renderer';
import Quotes from './Quotes';

it('renders correctly', () => {
  const tree = renderer
    .create(<Quotes />)
    .toTree();
  expect(tree).toMatchSnapshot();
});
