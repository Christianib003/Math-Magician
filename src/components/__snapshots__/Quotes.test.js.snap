// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Object {
  "instance": null,
  "nodeType": "component",
  "props": Object {},
  "rendered": Object {
    "instance": null,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <p
          className="qtext"
        >
          1. “All our dreams can come true if we have the courage to pursue them.”
          <span>
            Walt Disney
          </span>
        </p>,
        <p
          className="qtext"
        >
          2. “If you hear a voice within you say ‘you cannot paint,’ then by all means paint and that voice will be silenced.”
          <span>
            Vincent Van Gogh
          </span>
        </p>,
        <p
          className="qtext"
        >
          3. “It does not matter how slowly you go, so long as you do not stop.”
          <span>
            Confucius
          </span>
        </p>,
      ],
      "className": "quotes",
    },
    "rendered": Array [
      Object {
        "instance": null,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "1. “All our dreams can come true if we have the courage to pursue them.”",
            <span>
              Walt Disney
            </span>,
          ],
          "className": "qtext",
        },
        "rendered": Array [
          "1. “All our dreams can come true if we have the courage to pursue them.”",
          Object {
            "instance": null,
            "nodeType": "host",
            "props": Object {
              "children": "Walt Disney",
            },
            "rendered": Array [
              "Walt Disney",
            ],
            "type": "span",
          },
        ],
        "type": "p",
      },
      Object {
        "instance": null,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "2. “If you hear a voice within you say ‘you cannot paint,’ then by all means paint and that voice will be silenced.”",
            <span>
              Vincent Van Gogh
            </span>,
          ],
          "className": "qtext",
        },
        "rendered": Array [
          "2. “If you hear a voice within you say ‘you cannot paint,’ then by all means paint and that voice will be silenced.”",
          Object {
            "instance": null,
            "nodeType": "host",
            "props": Object {
              "children": "Vincent Van Gogh",
            },
            "rendered": Array [
              "Vincent Van Gogh",
            ],
            "type": "span",
          },
        ],
        "type": "p",
      },
      Object {
        "instance": null,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "3. “It does not matter how slowly you go, so long as you do not stop.”",
            <span>
              Confucius
            </span>,
          ],
          "className": "qtext",
        },
        "rendered": Array [
          "3. “It does not matter how slowly you go, so long as you do not stop.”",
          Object {
            "instance": null,
            "nodeType": "host",
            "props": Object {
              "children": "Confucius",
            },
            "rendered": Array [
              "Confucius",
            ],
            "type": "span",
          },
        ],
        "type": "p",
      },
    ],
    "type": "div",
  },
  "type": [Function],
}
`;
